apps/backend/prisma/schema.prisma

```diff
@@ -13,7 +13,68 @@ datasource db {
   url      = env("DATABASE_URL")
 }
 
+enum Priority {
+  LOW
+  MEDIUM
+  HIGH
+}
+
+enum Status {
+  OPEN
+  IN_PROGRESS
+  DONE
+}
```

Comment:
Esetleg még felvenném a Waiting for review statust, beadandoknal nagyon jól jön

Reply:
Illetve az open helyett a not started talan jobban illik ilyesmi assignmentekhez, tudom ez megjelenites, csak akkor konzisztens

Reply:
Jo otlet